<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterinstructionDialog</class>
 <widget class="QDialog" name="ajouterinstructionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type d'instruction</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioSimple">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Instruction simple</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioComposee">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Instruction compos√©e (groupe)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnValider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnnuler">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Contenu</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditContenu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>441</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(texte d'aide dynamique)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
